lib/actions/authActions.js

```diff
@@ -54,20 +54,15 @@ export function signup(newUserData) {
   };
 }
 
-export function validateToken() {
-  const {accessToken, uid, client} = JSON.parse(storage.get('ffc-token'));
+export function validateToken({accessToken, uid, client}) {
   return {
     types: [VALIDATE_TOKEN, VALIDATE_TOKEN_SUCCESS, VALIDATE_TOKEN_FAIL],
     promise: _client => _client.get('/auth/validate_token', {
-      params: {
+      data: {
```

Comment:
i'll accept this as data but i think it has to be params, again i'll double check in postman